aptos-move/framework/aptos-framework/sources/TokenV1.move

```diff
@@ -229,9 +229,9 @@ module AptosFramework::TokenV1 {
         royalty_points_denominator: u64,
         royalty_points_nominator: u64,
         token_mutate_setting: vector<bool>,
-        property_keys: vector<String>,
```

Comment:
Just curious, this did not work where? (using String here?)

In the VM restrictions like this have been removed. And with the upcoming utf8 strings, ideally we can use them also for scripts.

Reply:
Our adapter doesn't allow string as a parameter for script function now.  that's great.  we can then relax the checking in adapter.